templates/frontend/objects/monograph_full.tpl

```diff
@@ -390,13 +390,13 @@
 						{assign var=publicationDates value=$publicationFormat->getPublicationDates()}
 						{assign var=publicationDates value=$publicationDates->toArray()}
 						{assign var=hasPubId value=false}
```

Comment:
@NateWr, could you review this PR with an eye to this code for presenting various kinds of IDs in the front end? It looks to me like it wasn't working as expected -- `hasPubId` appears never to be set/used, and I think we're probably presenting publication format identifiers in places where we should be presenting book identifiers.